OxfordBodleian/Juel-Jensen/BDLaethf32.xml

```diff
@@ -228,51 +229,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                    <desc>Crude drawings of human figures in red ink or blue pencil.</desc> 
+                                    <desc>Crude drawings of human figures in red ink or blue pencil.</desc>
                                 </item>
                                 <item xml:id="e6">
                                     <locus from="81v" to="82r"/>
                                     <desc>Several names of individuals have been added in blue pencil or violet ink in a very inaccurate handwriting.
-                                        The names of <persName ref="PRS13885GabraHe"/> (spelled once as Gabra Ḥaywat), <persName ref="PRS13886GabraIy"/> (spelled once as Gabra ʾƎyasus), 
-                                        <persName ref="PRS13887WaldaGa"/> (spelled as Walda Garʾel), and <persName ref="PRS13888WalattaGa"/> are still legible.</desc> 
+                                        The names of <persName ref="PRS13885GabraHe"/> (spelled once as Gabra Ḥaywat), <persName ref="PRS13886GabraIy"/> (spelled once as Gabra ʾƎyasus),
+                                        <persName ref="PRS13887WaldaGa"/> (spelled as Walda Garʾel), and <persName ref="PRS13888WalattaGa"/> are still legible.</desc>
                                 </item>
                             </list>
                         </additions>
-                    
+
                         <bindingDesc>
-                            <binding contemporary="true">
-                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards.
-                                    The wood of the boards was originally employed for another purpose and was later reused to obtain the boards for binding.
-                                    On the back board the following writing <q xml:lang="en">"Profumeria 20555 Extra Mil"</q> (?) is still legible, together with decorative interlaces.
-                                    Part of the writing is now lost.
+                            <binding contemporary="Ethiopian">
+                                <decoNote xml:id="b1" type="bindingMaterial">
+                                  The book is bound in <material key="wood">wooden</material> boards.
                                 </decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                                    <material key="wood"/>       
+                                <decoNote xml:id="b2" type="Boards">
+                                  The boards are flush with the bookblock and have shallow external bevels. The grain runs vertically.
+                                  The external surface of the boards is polished, while the inner surface exhibits signs of the working tool.
+                                  The wood of the boards was originally employed for another purpose and was later reused to obtain the boards for binding.
```

Comment:
Maybe the reader will ask him/herself what "another purpose" means and especially how we know it.